services/common/src/redux/slices/mineReportPermitRequirementSlice.ts

```diff
@@ -27,11 +27,16 @@ const mineReportPermitRequirementSlice = createAppSlice({
         const headers = createRequestHeader();
         thunkApi.dispatch(showLoading());
 
+        const formatPayload = {
+          ...payload.values,
+          permit_condition_ids: [payload.values.permit_condition_id]
```

Comment:
just the minimum to run the post request- will most likely change when starting on FE